pallets/order-book/src/lib.rs

```diff
@@ -414,6 +416,11 @@ pub mod pallet {
                 order_book_id.base != order_book_id.quote,
                 Error::<T>::ForbiddenToCreateOrderBookWithSameAssets
             );
+            // https://github.com/sora-xor/sora2-network/issues/536
```

Comment:
comment is redundamt

Reply:
I thought it could explain why this check is needed (it's not obvious for a person without a context of this problem)
Now that I think about it, `git blame` should show this PR, so it should be fine without the comment, thanks :)